zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/SessionsThreadLocal.java

```diff
@@ -0,0 +1,10 @@
+package com.github.onsdigital.zebedee.session.store;
+
+import com.github.onsdigital.impl.UserDataPayload;
+
+import javax.servlet.http.HttpServletRequest;
+
+public interface SessionsThreadLocal {
+
+    void store(HttpServletRequest request, String secretKey) throws Exception;
```

Comment:
This should throw a specific Exception type. 

Lets create a new exception class `SessionStoreException` and then update this method to throw one of those instead.